src/llvm/lib_llvm/core.cr

```diff
@@ -25,6 +25,7 @@ lib LibLLVM
   fun get_enum_attribute_kind_for_name = LLVMGetEnumAttributeKindForName(name : Char*, s_len : SizeT) : UInt
   fun get_last_enum_attribute_kind = LLVMGetLastEnumAttributeKind : UInt
   fun create_enum_attribute = LLVMCreateEnumAttribute(c : ContextRef, kind_id : UInt, val : UInt64) : AttributeRef
+  fun create_string_attribute = LLVMCreateStringAttribute(c : ContextRef, k : Char*, k_length : UInt32, v : Char*, v_length : UInt32) : AttributeRef
```

Comment:
The split headers no longer use `UInt32` directly, unless the source header uses `uint32_t`:

```suggestion
  fun create_string_attribute = LLVMCreateStringAttribute(c : ContextRef, k : Char*, k_length : UInt, v : Char*, v_length : UInt) : AttributeRef
```

Reply:
Fixed (sorry this took a hot minute).